stable/yugaware/templates/statefulset.yaml

```diff
@@ -130,7 +146,7 @@ spec:
           {{- end }}
           command: [ "/bin/bash", "-c"]
           args:
-            - "cp /default_prometheus_config/prometheus.yml /prometheus_configs/prometheus.yml && bin/yugaware -Dconfig.file=/data/application.docker.conf"
```

Comment:
So we don't need [this](https://github.com/yugabyte/charts/pull/63/files#diff-58c816a5badea1634d1f7fd6bc835152552ed86e143de953e41b0953a9313deeR197-R198) anymore, right ?

Reply:
I guess this is something I missed while rebasing to remove the merge conflicts. Indeed, I'll remove the `volumeMount` related to default Prometheus conf in YW STS Pod Spec.